### PR TITLE
Test done 

### PR DESCRIPTION
**Pull request by olehmarushchak**
npm test not working!

- [DEMO LINK](https://olehmarushchak.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://olehmarushchak.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] **CSS Variable** is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
```
Comment (volodymyr-soltys97) [inline src/index.html:14]:
```suggestion
    <link
     rel="preconnect"
     href="https://fonts.gstatic.com"
     crossorigin="anonymous"
   >
```

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,54 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="fix">
+        <img
+         src="images\logo.png"
+         alt="logo"
+         class="logo_img">
```
Comment (volodymyr-soltys97) [inline src/index.html:23]:
```suggestion
        <img
         src="images\logo.png"
         alt="logo"
         class="logo_img"
       >
```

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,54 @@
...
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li>
```
Comment (volodymyr-soltys97) [inline src/index.html:28]: Add this classes everywhere
```suggestion
          <li class="nav__item">
```

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,54 @@
...
+            <a href="index.html" class="nav__link is-active">apple</a>
+          </li>
+          <li>
```
Comment (volodymyr-soltys97) [inline src/index.html:31]: Add the blank line between the links
```suggestion
          </li>
          
          <li>
```

File: src/index.html [hunk 5]
```diff
@@ -7,8 +7,54 @@
...
+            <a href="index.html" class="nav__link">smartphones</a>
+          </li>
+          <li>
+            <a
+             href="index.html"
+             class="nav__link"
+             data-qa="hover">laptops & computers
+            </a>
```
Comment (volodymyr-soltys97) [inline src/index.html:42]:
```suggestion
            <a
              href="index.html"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, serif;
+  background-color: aqua;
+  margin: 0;
+  padding: 0;
+
+  --main-color: #00ACDC;
```
Comment (volodymyr-soltys97) [inline src/style.css:7]: Create the variable with root selector

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,73 @@
...
+}
+
+.is-active {
+  color: var(--main-color);
+  position: relative;
+}
+
+.fix {
+  font-size: 0;
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:72]:
```suggestion
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,73 @@
...
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
+  padding: 0 50px;
+}
+
+ul.nav_list li {
```
Comment (volodymyr-soltys97) [inline src/style.css:23]: Use the classes selector here

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
1. Try to fix the test report link
image

2. Need to remove the left padding for lists
image

3. Remove the left margin for the first link
image

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,74 @@
...
+  font-size: 12px;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  color: black;
+  height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:39]: it's no a good practice to hardcode height for elements with content inside. use padding instead

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,74 @@
...
+  text-decoration: none;
+  margin-left: 10px;
```
Comment (Viktor-Kost) [inline src/style.css:41]: You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,74 @@
...
+  height: 60px;
+  padding: 0 50px;
+}
+
+ul.nav_list li {
+  display: inline-flex;
+}
+
+.nav_list {
+  line-height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:28]:
```suggestion
```

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,74 @@
...
+  font-weight: bold;
+  font-size: 12px;
+  text-transform: uppercase;
```
Comment (Viktor-Kost) [inline src/style.css:31]: move font styles to a link

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,74 @@
...
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:19]:
```suggestion
```

File: src/index.html [hunk 14]
```diff
@@ -7,8 +7,67 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="index.html" class="nav__link">video</a>
+          </li>
+
```
Comment (Viktor-Kost) [inline src/index.html:68]:
```suggestion
```

File: src/index.html [hunk 15]
```diff
@@ -7,8 +7,67 @@
...
+        <img
+         src="images\logo.png"
+         alt="logo"
+         class="logo_img"
+         >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+
```
Comment (Viktor-Kost) [inline src/index.html:30]:
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
check previous mentor's remarks 
https://github.com/mate-academy/layout_moyo-header/pull/4415#pullrequestreview-1836177748 (point 3)
https://github.com/mate-academy/layout_moyo-header/pull/4415#discussion_r1461681667
https://github.com/mate-academy/layout_moyo-header/pull/4415#discussion_r1461681972
https://github.com/mate-academy/layout_moyo-header/pull/4415#discussion_r1461683825
https://github.com/mate-academy/layout_moyo-header/pull/4415#discussion_r1461684143
https://github.com/mate-academy/layout_moyo-header/pull/4415#discussion_r1461684406
https://github.com/mate-academy/layout_moyo-header/pull/4415#discussion_r1461684655

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls fix prev comments
Pls ask for help in **fe_chat** if you don't understand something

**Pull request closed without merging**